### ISSUE TITLE
New Collection not aligned with the rest of the content

### ISSUE THREAD

**Issue by lukeromanowicz** [DivanteLtd/vue-storefront#2556]
## Current behavior
The button responsible for opening and closing menu is not aligned with New Collection section in desktop view:
![image](https://user-images.githubusercontent.com/46789227/53945632-4fe8ca00-40c2-11e9-967d-988129b9da9c.png)

## Expected behavior
New Collection section is aligned with menu hamburger and the rest of the page

## Steps to reproduce the issue
Open homepage in desktop mode

# Can you handle fixing this bug by yourself?
Yes but not in the nearest future. Feel free to propose a PR

## Environment details
- Browser: Chrome 72
- OS: Ubuntu
- Node: The same that is set in docker

**Comment by RGijsberts** [comment]
I believe the hamburger menu icon is aligned correctly. It aligns with the New Collection section and the Get Inspired section on the Demo homepage. I also tested this with other elements on other pages and it seems to be aligned correctly. 

The issue seems to be related to the Offers section on the demo homepage. This section is not aligned correctly, see the screenshot I added.

![screenshot 2019-03-08 at 10 40 15](https://user-images.githubusercontent.com/7312191/54021715-0d41f300-4191-11e9-9615-69c4cad8d9e4.png)
